### PR TITLE
Add some fixes to the Popover for Vue 2.0

### PR DESCRIPTION
**Pull request by ShirajG**
This PR adds some basic support for using the Popover with Vue 2.0

### CODE REVIEW

File: src/Popover.vue [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-    :class="['popover',placement]"
-    :transition="effect"
-  >
-    <div class="arrow"></div>
-    <h3 class="popover-title" v-if="title">
-      <slot name="title">{{title}}</slot>
-    </h3>
-    <div class="popover-content">
-      <slot name="content">{{{content}}}</slot>
+  <span>
```
Comment (ShirajG) [inline src/Popover.vue]: Added a containing element, Vue 2.0 no longer supports fragments.
http://vuejs.org/guide/migration.html#Fragment-Instances-deprecated

File: src/Popover.vue [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+    <span ref="trigger">
```
Comment (ShirajG) [inline src/Popover.vue]: Use the new ref attribute

File: src/Popover.vue [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
+    </span>
+    <div ref="popover" v-show="show"
+      :class="['popover',placement]"
+      :transition="effect"
+    >
+      <div class="arrow"></div>
+      <h3 class="popover-title" v-if="title">
+        <slot name="title">{{title}}</slot>
+      </h3>
+      <div v-html="content" class="popover-content">
```
Comment (ShirajG) [inline src/Popover.vue]: HTML interploation syntax deprecated no longer supported in Vue 2.

**Comment by ShirajG** [comment]
This is some work I've done towards making a piece of this library compatible with vue 2.0.
Is there a branch for that I should submit this to?

**Comment by mediaessenz** [comment]
Your modification is not working with current v2 branch.
No popover appear after apply.
Do I have to change any markup to get it working?

**Comment by ShirajG** [comment]
Ah, if it's working in the current 2.0 branch this is not necessary, I had made this a few days after the Vue 2 release, the v2 branch was not in existence IIRC.

**Comment by wffranco** [comment]
yuche v2 is the branch for vue2.
but if you want to be updated check my fork

**Pull request closed without merging**